src/types/durableClientTypes.ts

```diff
@@ -1,9 +1,229 @@
-import { FunctionInput } from "@azure/functions";
+import { FunctionInput, HttpRequest, HttpResponse } from "@azure/functions";
+import { EntityId, EntityStateResponse, HttpManagementPayload, IHttpResponse } from "../classes";
+import { DurableOrchestrationStatus } from "../durableorchestrationstatus";
+import { IHttpRequest } from "../ihttprequest";
+import { OrchestrationRuntimeStatus } from "../orchestrationruntimestatus";
+import { PurgeHistoryResult } from "../purgehistoryresult";
 
 export interface DurableClientInput extends FunctionInput {
     type: "durableClient";
 }
 
+/**
+ * Client for starting, querying, terminating and raising events to
+ * orchestration instances.
```

Comment:
nitpick: also for entities
```suggestion
 * orchestration and entities.
```

Reply:
Done